jinja_utils.py

```diff
@@ -25,6 +25,9 @@
 import json
 
 
+_OPPIA_ROOT_FILEPATH = 'app.js'
```

Comment:
I think this variable name isn't describing what it contains. Perhaps '_OPPIA_MODULE_DEFINITION_FILE' might be more appropriate. WDYT?